### ISSUE TITLE
Sonoma 14.1  on new MacBook Pro with lots of RAM - QS 2.4.1 gets unresponsive 8-10 times daily

### ISSUE THREAD

**Issue by alland138** [quicksilver/Quicksilver#2988]
### Before submitting your bug report, please confirm you have completed the following steps

- [X] Searched the issue is not already listed in the [Issue List](https://github.com/quicksilver/Quicksilver/issues?q=is%3Aissue) (I've searched it)
- [X] Searched in the [Quicksilver Users' Group](https://groups.google.com/forum/#!forum/blacktree-quicksilver) for questions relating to tech support.
- [X] Downloaded the [latest version of Quicksilver](https://qsapp.com/) for your system and confirmed the issue still persists.

### Bug description

There’s no apparent pattern or reproducible cause but QS just seems to freeze multiple times a day (i.e. unresponsive). Only way forward is to force quit and restart QS. Never had a problem with QS until I started running Sonoma - and I have no idea what’s causing this - there are no crashes generating logs or messages

### Steps to reproduce

Running the computer and doing normal work — nothing that apparently triggers the unresponsive state

### Expected behavior

I expect QS to run smoothly, as it has always run in the past

### MacOS Version

Other

### Quicksilver Version

2.4.1

### Relevant Plugins

None

### Crash Logs or Spindump

If you let me know what you’d like me to do i.e. describe steps you need taken, I’ll be glad to do it

### Additional info

Running Sonoma 14.1

**Comment by alland138** [comment]
I now suddenly usually have the crash reporting come up so I send in the crash log - here is the latest one:

-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Quicksilver [20484]
Path:                  /Applications/Quicksilver.app/Contents/MacOS/Quicksilver
Identifier:            com.blacktree.Quicksilver
Version:               2.4.1 (4040)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2023-11-14 11:23:03.9620 +0100
OS Version:            macOS 14.1.1 (23B81)
Report Version:        12
Anonymous UUID:        C5BA9526-6014-0853-CFD1-7D45DD991E3D

Sleep/Wake UUID:       F843AD94-BEE2-42F7-9BD1-22844E1457AD

Time Awake Since Boot: 65000 seconds
Time Since Wake:       15566 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Termination Reason:    Namespace SIGNAL, Code 6 Abort trap: 6
Terminating Process:   Quicksilver [20484]

Application Specific Information:
abort() called


Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	       0x18912511c __pthread_kill + 8
1   libsystem_pthread.dylib       	       0x18915ccc0 pthread_kill + 288
2   libsystem_c.dylib             	       0x18906ca40 abort + 180
3   libc++abi.dylib               	       0x1891146d8 abort_message + 132
4   libc++abi.dylib               	       0x1891047c8 demangling_terminate_handler() + 348
5   libobjc.A.dylib               	       0x188daf894 _objc_terminate() + 144
6   libc++abi.dylib               	       0x189113a9c std::__terminate(void (*)()) + 16
7   libc++abi.dylib               	       0x189113a40 std::terminate() + 56
8   libdispatch.dylib             	       0x188faf924 _dispatch_client_callout + 40
9   libdispatch.dylib             	       0x188fbdfa8 _dispatch_main_queue_drain + 984
10  libdispatch.dylib             	       0x188fbdbc0 _dispatch_main_queue_callback_4CF + 44
11  CoreFoundation                	       0x18927b15c __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 16
12  CoreFoundation                	       0x189238a80 __CFRunLoopRun + 1996
13  CoreFoundation                	       0x189237c5c CFRunLoopRunSpecific + 608
14  HIToolbox                     	       0x1937b4448 RunCurrentEventLoopInMode + 292
15  HIToolbox                     	       0x1937b4284 ReceiveNextEventCommon + 648
16  HIToolbox                     	       0x1937b3fdc _BlockUntilNextEventMatchingListInModeWithFilter + 76
17  AppKit                        	       0x18ca12c54 _DPSNextEvent + 660
18  AppKit                        	       0x18d1e8ebc -[NSApplication(NSEventRouting) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 716
19  AppKit                        	       0x18ca06100 -[NSApplication run] + 476
20  AppKit                        	       0x18c9dd3cc NSApplicationMain + 880
21  dyld                          	       0x188de10e0 start + 2360

Thread 1:: VDKQueue Watcher Thread
0   libsystem_kernel.dylib        	       0x189122a00 kevent + 8
1   QSCore                        	       0x104cd22d4 -[VDKQueue watcherThread:] + 260
2   Foundation                    	       0x18a339ff4 __NSThread__start__ + 716
3   libsystem_pthread.dylib       	       0x18915d034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x189157e3c thread_start + 8

Thread 2:: com.apple.NSEventThread
0   libsystem_kernel.dylib        	       0x18911c8b4 mach_msg2_trap + 8
1   libsystem_kernel.dylib        	       0x18912ed30 mach_msg2_internal + 80
2   libsystem_kernel.dylib        	       0x1891254f0 mach_msg_overwrite + 476
3   libsystem_kernel.dylib        	       0x18911cc38 mach_msg + 24
4   CoreFoundation                	       0x189239ea4 __CFRunLoopServiceMachPort + 160
5   CoreFoundation                	       0x18923876c __CFRunLoopRun + 1208
6   CoreFoundation                	       0x189237c5c CFRunLoopRunSpecific + 608
7   AppKit                        	       0x18cb3c404 _NSEventThread + 144
8   libsystem_pthread.dylib       	       0x18915d034 _pthread_start + 136
9   libsystem_pthread.dylib       	       0x189157e3c thread_start + 8

Thread 3:
0   libsystem_pthread.dylib       	       0x189157e28 start_wqthread + 0

Thread 4::  Dispatch queue: QSCatalogEntry scanQueue: QSPresetDesktop
0   libsystem_kernel.dylib        	       0x18911f564 __psynch_mutexwait + 8
1   libsystem_pthread.dylib       	       0x18915a168 _pthread_mutex_firstfit_lock_wait + 84
2   libsystem_pthread.dylib       	       0x189157af8 _pthread_mutex_firstfit_lock_slow + 248
3   CarbonCore                    	       0x18c50dd1c TSLockMutex + 176
4   CarbonCore                    	       0x18c544600 GetRefHandle + 36
5   CarbonCore                    	       0x18c5488c0 CloseResFileCommon + 320
6   QSFoundation                  	       0x104aa94e8 -[NDResourceFork closeFile] + 32
7   QSFoundation                  	       0x104aa8de0 -[NDAlias(AliasFile) initWithContentsOfFile:] + 116
8   QSFoundation                  	       0x104aa9190 +[NDAlias(AliasFile) aliasWithContentsOfFile:] + 36
9   Core Support                  	       0x10f53f630 0x10f508000 + 226864
10  Core Support                  	       0x10f53ef80 0x10f508000 + 225152
11  Core Support                  	       0x10f541238 0x10f508000 + 234040
12  QSCore                        	       0x104c8b684 -[QSCatalogEntry scannedObjects] + 76
13  QSCore                        	       0x104c8c508 __30-[QSCatalogEntry scanAndCache]_block_invoke + 32
14  libdispatch.dylib             	       0x188faf910 _dispatch_client_callout + 20
15  libdispatch.dylib             	       0x188fbecc4 _dispatch_lane_barrier_sync_invoke_and_complete + 56
16  QSCore                        	       0x104c8b8bc -[QSCatalogEntry scanAndCache] + 208
17  QSCore                        	       0x104c8bbe8 -[QSCatalogEntry scanForced:] + 416
18  CoreFoundation                	       0x18922e820 __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 148
19  CoreFoundation                	       0x1892c28ec ___CFXRegistrationPost_block_invoke + 88
20  CoreFoundation                	       0x1892c2834 _CFXRegistrationPost + 440
21  CoreFoundation                	       0x1891fd48c _CFXNotificationPost + 764
22  Foundation                    	       0x18a2ef030 -[NSNotificationCenter postNotificationName:object:userInfo:] + 88
23  QSCore                        	       0x104cd1268 -[QSVoyeur queue:didReceiveNotification:forPath:] + 196
24  QSCore                        	       0x104cd29e0 __26-[VDKQueue watcherThread:]_block_invoke + 228
25  libdispatch.dylib             	       0x188fadcb8 _dispatch_call_block_and_release + 32
26  libdispatch.dylib             	       0x188faf910 _dispatch_client_callout + 20
27  libdispatch.dylib             	       0x188fc13cc _dispatch_root_queue_drain + 864
28  libdispatch.dylib             	       0x188fc1a04 _dispatch_worker_thread2 + 156
29  libsystem_pthread.dylib       	       0x1891590d8 _pthread_wqthread + 228
30  libsystem_pthread.dylib       	       0x189157e30 start_wqthread + 8


Thread 0 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000000   x1: 0x0000000000000000   x2: 0x0000000000000000   x3: 0x0000000000000000
    x4: 0x0000000189118377   x5: 0x000000016b8fe2b0   x6: 0x000000000000006e   x7: 0x0000000000000860
    x8: 0xbf9bf3f4203edb45   x9: 0xbf9bf3f5ff0e0585  x10: 0x0000000000000200  x11: 0x000000000000000b
   x12: 0x0000000000000000  x13: 0x00000000001ff800  x14: 0x00000000000007fb  x15: 0x00000000fb62e025
   x16: 0x0000000000000148  x17: 0x00000001e82fbda0  x18: 0x0000000000000000  x19: 0x0000000000000006
   x20: 0x00000001df30dec0  x21: 0x0000000000000103  x22: 0x00000001df30dfa0  x23: 0x0000000000000114
   x24: 0x0000000000000000  x25: 0x00000001df30dfa0  x26: 0x0000600000aa5c40  x27: 0x000000000000000f
   x28: 0x0000000000000000   fp: 0x000000016b8fe220   lr: 0x000000018915ccc0
    sp: 0x000000016b8fe200   pc: 0x000000018912511c cpsr: 0x40001000
   far: 0x0000000000000000  esr: 0x56000080  Address size fault

Binary Images:
       0x10f784000 -        0x10f787fff com.blacktree.Quicksilver.QSPrimerInterfacePlugIn (1.3.0) <4c7e4541-181f-308c-bf52-e6cbdba6744a> /Applications/Quicksilver.app/Contents/PlugIns/Primer Interface.qsplugin/Contents/MacOS/Primer Interface
       0x10f770000 -        0x10f773fff com.blacktree.Quicksilver.QSHotKeyTriggerPlugIn (3.1.0) <aafd037b-b7c3-34b7-8cd9-85adc7ea8c21> /Applications/Quicksilver.app/Contents/PlugIns/Keyboard Triggers.qsplugin/Contents/MacOS/Keyboard Triggers
       0x10f4f0000 -        0x10f4f3fff com.apple.icloud.drive.fileprovider.override (1.0) <32076e3f-af4f-3832-9a49-6552ddbcac62> /System/Library/Frameworks/FileProvider.framework/OverrideBundles/iCloudDriveFileProviderOverride.bundle/Contents/MacOS/iCloudDriveFileProviderOverride
       0x10f638000 -        0x10f657fff com.apple.findersync.fileprovideroverride.FinderSyncCollaborationFileProviderOverride (14.1) <c69dfd08-9b3e-3e05-800d-c446e7e0502a> /System/Library/Frameworks/FileProvider.framework/OverrideBundles/FinderSyncCollaborationFileProviderOverride.bundle/Contents/MacOS/FinderSyncCollaborationFileProviderOverride
       0x10f354000 -        0x10f35bfff com.apple.FileProviderOverride (1703.41.2) <78f21b30-a3ca-3fc0-992f-a421a2e10051> /System/Library/Frameworks/FileProvider.framework/OverrideBundles/FileProviderOverride.bundle/Contents/MacOS/FileProviderOverride
       0x10f508000 -        0x10f563fff com.blacktree.Quicksilver.QSCorePlugIn (1.3.0) <fa0b4288-55e7-3abe-aa4c-ae482e55ebc6> /Applications/Quicksilver.app/Contents/PlugIns/Core Support.qsplugin/Contents/MacOS/Core Support
       0x10f374000 -        0x10f393fff com.apple.security.csparser (3.0) <af53e762-bef4-3d88-9797-ff28f3f85e76> /System/Library/Frameworks/Security.framework/Versions/A/PlugIns/csparser.bundle/Contents/MacOS/csparser
       0x1047a4000 -        0x1047affff libobjc-trampolines.dylib (*) <c9ce7923-2f2d-31eb-a6e4-0f3b7a78f895> /usr/lib/libobjc-trampolines.dylib
       0x104a8c000 -        0x104ac7fff com.blacktree.QSFoundation (*) <5ca56208-d6aa-3ede-a16c-2786258aa4eb> /Applications/Quicksilver.app/Contents/Frameworks/QSFoundation.framework/Versions/A/QSFoundation
       0x1049e8000 -        0x1049f3fff com.blacktree.QSEffects (*) <86b20182-7aa2-3072-81ff-01c5dbfa7096> /Applications/Quicksilver.app/Contents/Frameworks/QSEffects.framework/Versions/A/QSEffects
       0x104c80000 -        0x104cebfff com.blacktree.QSCore (*) <56d32f7c-fbe1-3740-9e71-3ae69cde15c1> /Applications/Quicksilver.app/Contents/Frameworks/QSCore.framework/Versions/A/QSCore
       0x104a08000 -        0x104a3ffff com.blacktree.QSInterface (*) <23bfea87-78c4-3726-8c2a-10954e3c4ba2> /Applications/Quicksilver.app/Contents/Frameworks/QSInterface.framework/Versions/A/QSInterface
       0x1049a8000 -        0x1049b7fff com.flyingmeat.FMDB-MacOS (2.7.7) <e8ff4f99-f16c-383f-87c9-10b6c4939472> /Applications/Quicksilver.app/Contents/Frameworks/QSFoundation.framework/Versions/A/Frameworks/FMDB.framework/Versions/A/FMDB
       0x104500000 -        0x10453bfff com.blacktree.Quicksilver (2.4.1) <9bdac77f-8ca1-3cb9-9c33-f570bc7a48f0> /Applications/Quicksilver.app/Contents/MacOS/Quicksilver
       0x18911b000 -        0x189155fef libsystem_kernel.dylib (*) <b7751381-1442-30b5-91b9-ad7be461bebe> /usr/lib/system/libsystem_kernel.dylib
       0x189156000 -        0x189162ff3 libsystem_pthread.dylib (*) <daf95373-5de6-39a1-a6ce-d87f3f0629cc> /usr/lib/system/libsystem_pthread.dylib
       0x188ff6000 -        0x189074ffb libsystem_c.dylib (*) <decb8685-f34a-3979-afcb-71fb55747e41> /usr/lib/system/libsystem_c.dylib
       0x189103000 -        0x18911afff libc++abi.dylib (*) <edbb6df0-72f9-3bc2-96fa-a86e05d3d305> /usr/lib/libc++abi.dylib
       0x188d8c000 -        0x188ddaf08 libobjc.A.dylib (*) <49e2dcb3-f014-3fcf-949b-f5f57b3ef0a8> /usr/lib/libobjc.A.dylib
       0x188fac000 -        0x188ff2fff libdispatch.dylib (*) <dc1d0187-7149-3100-bc63-f633afebee6c> /usr/lib/system/libdispatch.dylib
       0x1891bc000 -        0x189693fff com.apple.CoreFoundation (6.9) <9f046e36-7286-3a6e-a280-699d6e47cfaf> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
       0x193784000 -        0x193a43fff com.apple.HIToolbox (2.1.1) <4dffb7eb-bbcd-3ff6-a203-5a0b97358ee8> /System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/HIToolbox
       0x18c9d9000 -        0x18dcedfff com.apple.AppKit (6.9) <b475e541-691d-39c6-8ca4-e414b0d3dd88> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
       0x188ddb000 -        0x188e6f317 dyld (*) <ec7a3ba0-f9bf-3ab8-a0f4-8622e5606b20> /usr/lib/dyld
               0x0 - 0xffffffffffffffff ??? (*) <00000000-0000-0000-0000-000000000000> ???
       0x18a2e6000 -        0x18af33fff com.apple.Foundation (6.9) <2126fd9a-52ce-3eb2-8640-b57535e073b8> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
       0x18c50a000 -        0x18c80efff com.apple.CoreServices.CarbonCore (1333) <049ccd7b-4151-3816-8fa8-a0bad75ae69b> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/CarbonCore.framework/Versions/A/CarbonCore

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=1.4G resident=0K(0%) swapped_out_or_unallocated=1.4G(100%)
Writable regions: Total=2.2G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=2.2G(100%)

                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Accelerate framework               128K        1 
Activity Tracing                   256K        1 
ColorSync                          576K       27 
CoreAnimation                      384K       24 
CoreGraphics                        48K        3 
CoreServices                       624K        2 
CoreUI image data                  128K        2 
Foundation                          16K        1 
Image IO                           176K        1 
Kernel Alloc Once                   32K        1 
MALLOC                             2.1G       84 
MALLOC guard page                  288K       18 
Memory Tag 244                      16K        1 
SQLite page cache                  384K        3 
STACK GUARD                       56.1M        5 
Stack                             10.1M        5 
VM_ALLOCATE                        144K        9 
__AUTH                            1669K      332 
__AUTH_CONST                      27.5M      537 
__CTF                               824        1 
__DATA                            12.7M      536 
__DATA_CONST                      27.3M      556 
__DATA_DIRTY                      1705K      196 
__FONT_DATA                          4K        1 
__LINKEDIT                       887.5M       15 
__OBJC_RO                         70.8M        1 
__OBJC_RW                         2156K        1 
__TEXT                           590.7M      573 
dyld private memory                272K        2 
libnetwork                         640K       16 
mapped file                      183.7M       41 
shared memory                      896K       16 
===========                     =======  ======= 
TOTAL                              4.0G     3012 



-----------
Full Report
-----------

{"app_name":"Quicksilver","timestamp":"2023-11-14 11:23:04.00 +0100","app_version":"2.4.1","slice_uuid":"9bdac77f-8ca1-3cb9-9c33-f570bc7a48f0","build_version":"4040","platform":1,"bundleID":"com.blacktree.Quicksilver","share_with_app_devs":0,"is_first_party":0,"bug_type":"309","os_version":"macOS 14.1.1 (23B81)","roots_installed":0,"name":"Quicksilver","incident_id":"EA4FB537-2CC8-4B1A-885E-E50EA47763FE"}
{
  "uptime" : 65000,
  "procRole" : "Background",
  "version" : 2,
  "userID" : 501,
  "deployVersion" : 210,
  "modelCode" : "Mac14,6",
  "coalitionID" : 1268,
  "osVersion" : {
    "train" : "macOS 14.1.1",
    "build" : "23B81",
    "releaseType" : "User"
  },
  "captureTime" : "2023-11-14 11:23:03.9620 +0100",
  "codeSigningMonitor" : 1,
  "incident" : "EA4FB537-2CC8-4B1A-885E-E50EA47763FE",
  "pid" : 20484,
  "translated" : false,
  "cpuType" : "ARM-64",
  "roots_installed" : 0,
  "bug_type" : "309",
  "procLaunch" : "2023-11-13 22:52:45.4968 +0100",
  "procStartAbsTime" : 861189528170,
  "procExitAbsTime" : 1567538939519,
  "procName" : "Quicksilver",
  "procPath" : "\/Applications\/Quicksilver.app\/Contents\/MacOS\/Quicksilver",
  "bundleInfo" : {"CFBundleShortVersionString":"2.4.1","CFBundleVersion":"4040","CFBundleIdentifier":"com.blacktree.Quicksilver"},
  "storeInfo" : {"deviceIdentifierForVendor":"BC27C414-A0AF-5E8D-A309-D373F929F1D9","thirdParty":true},
  "parentProc" : "launchd",
  "parentPid" : 1,
  "coalitionName" : "com.blacktree.Quicksilver",
  "crashReporterKey" : "C5BA9526-6014-0853-CFD1-7D45DD991E3D",
  "codeSigningID" : "com.blacktree.Quicksilver",
  "codeSigningTeamID" : "ATVCND8EAP",
  "codeSigningFlags" : 570495745,
  "codeSigningValidationCategory" : 6,
  "codeSigningTrustLevel" : 4294967295,
  "instructionByteStream" : {"beforePC":"fyMD1f17v6n9AwCRXOD\/l78DAJH9e8Go\/w9f1sADX9YQKYDSARAA1A==","atPC":"AwEAVH8jA9X9e7+p\/QMAkVHg\/5e\/AwCR\/XvBqP8PX9bAA1\/WcAqA0g=="},
  "wakeTime" : 15566,
  "sleepWakeUUID" : "F843AD94-BEE2-42F7-9BD1-22844E1457AD",
  "sip" : "enabled",
  "exception" : {"codes":"0x0000000000000000, 0x0000000000000000","rawCodes":[0,0],"type":"EXC_CRASH","signal":"SIGABRT"},
  "termination" : {"flags":0,"code":6,"namespace":"SIGNAL","indicator":"Abort trap: 6","byProc":"Quicksilver","byPid":20484},
  "asi" : {"libsystem_c.dylib":["abort() called"]},
  "exceptionReason" : {"arguments":["__NSCFArray","objectAtIndex:","7","","2"],"format_string":"-[%s %s]: index (%ld%s) beyond bounds (%ld)","name":"NSInvalidArgumentException","type":"objc-exception","composed_message":"-[__NSCFArray objectAtIndex:]: index (7) beyond bounds (2)","class":"NSException"},
  "extMods" : {"caller":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"system":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"targeted":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"warnings":0},
  "lastExceptionBacktrace" : [{"imageOffset":993268,"symbol":"__exceptionPreprocess","symbolLocation":164,"imageIndex":20},{"imageOffset":106164,"symbol":"objc_exception_throw","symbolLocation":60,"imageIndex":18},{"imageOffset":1995052,"symbol":"_NSArrayRaiseInsertNilException","symbolLocation":0,"imageIndex":20},{"imageOffset":238776,"symbol":"-[__NSCFArray objectAtIndex:]","symbolLocation":200,"imageIndex":20},{"imageOffset":239140,"symbol":"GetRefHandle","symbolLocation":72,"imageIndex":26},{"imageOffset":238320,"symbol":"vCheckLoad","symbolLocation":80,"imageIndex":26},{"imageOffset":183488,"symbol":"GetResourceCommon","symbolLocation":176,"imageIndex":26},{"imageOffset":121960,"symbol":"operateOnResourceUsingFunction","symbolLocation":248,"imageIndex":8},{"imageOffset":120580,"symbol":"-[NDResourceFork dataForType:Id:]","symbolLocation":44,"imageIndex":8},{"imageOffset":48868,"symbol":"+[NSPasteboard(Clippings) pasteboardByFilteringClipping:]","symbolLocation":164,"imageIndex":8},{"imageOffset":165580,"symbol":"-[QSObject(Pasteboard) addContentsOfClipping:]","symbolLocation":36,"imageIndex":10},{"imageOffset":7352,"symbol":"_dispatch_call_block_and_release","symbolLocation":32,"imageIndex":19},{"imageOffset":14608,"symbol":"_dispatch_client_callout","symbolLocation":20,"imageIndex":19},{"imageOffset":73640,"symbol":"_dispatch_main_queue_drain","symbolLocation":984,"imageIndex":19},{"imageOffset":72640,"symbol":"_dispatch_main_queue_callback_4CF","symbolLocation":44,"imageIndex":19},{"imageOffset":782684,"symbol":"__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__","symbolLocation":16,"imageIndex":20},{"imageOffset":510592,"symbol":"__CFRunLoopRun","symbolLocation":1996,"imageIndex":20},{"imageOffset":506972,"symbol":"CFRunLoopRunSpecific","symbolLocation":608,"imageIndex":20},{"imageOffset":197704,"symbol":"RunCurrentEventLoopInMode","symbolLocation":292,"imageIndex":21},{"imageOffset":197252,"symbol":"ReceiveNextEventCommon","symbolLocation":648,"imageIndex":21},{"imageOffset":196572,"symbol":"_BlockUntilNextEventMatchingListInModeWithFilter","symbolLocation":76,"imageIndex":21},{"imageOffset":236628,"symbol":"_DPSNextEvent","symbolLocation":660,"imageIndex":22},{"imageOffset":8453820,"symbol":"-[NSApplication(NSEventRouting) _nextEventMatchingEventMask:untilDate:inMode:dequeue:]","symbolLocation":716,"imageIndex":22},{"imageOffset":184576,"symbol":"-[NSApplication run]","symbolLocation":476,"imageIndex":22},{"imageOffset":17356,"symbol":"NSApplicationMain","symbolLocation":880,"imageIndex":22},{"imageOffset":24800,"symbol":"start","symbolLocation":2360,"imageIndex":23}],
  "faultingThread" : 0,
  "threads" : [{"triggered":true,"id":601095,"threadState":{"x":[{"value":0},{"value":0},{"value":0},{"value":0},{"value":6594593655},{"value":6099559088},{"value":110},{"value":2144},{"value":13806897312472947525},{"value":13806897320506033541},{"value":512},{"value":11},{"value":0},{"value":2095104},{"value":2043},{"value":4217561125},{"value":328},{"value":8190410144},{"value":0},{"value":6},{"value":8039489216,"symbolLocation":0,"symbol":"_main_thread"},{"value":259},{"value":8039489440,"symbolLocation":224,"symbol":"_main_thread"},{"value":276},{"value":0},{"value":8039489440,"symbolLocation":224,"symbol":"_main_thread"},{"value":105553127431232},{"value":15},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":6594874560},"cpsr":{"value":1073745920},"fp":{"value":6099558944},"sp":{"value":6099558912},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6594646300,"matchesCrashFrame":1},"far":{"value":0}},"queue":"com.apple.main-thread","frames":[{"imageOffset":41244,"symbol":"__pthread_kill","symbolLocation":8,"imageIndex":14},{"imageOffset":27840,"symbol":"pthread_kill","symbolLocation":288,"imageIndex":15},{"imageOffset":485952,"symbol":"abort","symbolLocation":180,"imageIndex":16},{"imageOffset":71384,"symbol":"abort_message","symbolLocation":132,"imageIndex":17},{"imageOffset":6088,"symbol":"demangling_terminate_handler()","symbolLocation":348,"imageIndex":17},{"imageOffset":145556,"symbol":"_objc_terminate()","symbolLocation":144,"imageIndex":18},{"imageOffset":68252,"symbol":"std::__terminate(void (*)())","symbolLocation":16,"imageIndex":17},{"imageOffset":68160,"symbol":"std::terminate()","symbolLocation":56,"imageIndex":17},{"imageOffset":14628,"symbol":"_dispatch_client_callout","symbolLocation":40,"imageIndex":19},{"imageOffset":73640,"symbol":"_dispatch_main_queue_drain","symbolLocation":984,"imageIndex":19},{"imageOffset":72640,"symbol":"_dispatch_main_queue_callback_4CF","symbolLocation":44,"imageIndex":19},{"imageOffset":782684,"symbol":"__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__","symbolLocation":16,"imageIndex":20},{"imageOffset":510592,"symbol":"__CFRunLoopRun","symbolLocation":1996,"imageIndex":20},{"imageOffset":506972,"symbol":"CFRunLoopRunSpecific","symbolLocation":608,"imageIndex":20},{"imageOffset":197704,"symbol":"RunCurrentEventLoopInMode","symbolLocation":292,"imageIndex":21},{"imageOffset":197252,"symbol":"ReceiveNextEventCommon","symbolLocation":648,"imageIndex":21},{"imageOffset":196572,"symbol":"_BlockUntilNextEventMatchingListInModeWithFilter","symbolLocation":76,"imageIndex":21},{"imageOffset":236628,"symbol":"_DPSNextEvent","symbolLocation":660,"imageIndex":22},{"imageOffset":8453820,"symbol":"-[NSApplication(NSEventRouting) _nextEventMatchingEventMask:untilDate:inMode:dequeue:]","symbolLocation":716,"imageIndex":22},{"imageOffset":184576,"symbol":"-[NSApplication run]","symbolLocation":476,"imageIndex":22},{"imageOffset":17356,"symbol":"NSApplicationMain","symbolLocation":880,"imageIndex":22},{"imageOffset":24800,"symbol":"start","symbolLocation":2360,"imageIndex":23}]},{"id":601132,"name":"VDKQueue Watcher Thread","threadState":{"x":[{"value":4},{"value":0},{"value":0},{"value":6102412504},{"value":1},{"value":6102412480},{"value":105553171696832},{"value":6102409816},{"value":1},{"value":0},{"value":0},{"value":8039505200,"symbolLocation":496,"symbol":"_dispatch_root_queues"},{"value":105553135776552},{"value":2095104},{"value":2043},{"value":3210932349},{"value":363},{"value":2},{"value":0},{"value":105553171696832},{"value":105553122334528},{"value":105553149323824},{"value":3},{"value":7916785716,"objc-selector":"path"},{"value":7916757191,"objc-selector":"respondsToSelector:"},{"value":65532},{"value":7916770017,"objc-selector":"addObject:"},{"value":105553136173232},{"value":105553133205792}],"flavor":"ARM_THREAD_STATE64","lr":{"value":4375519956},"cpsr":{"value":1610616832},"fp":{"value":6102412640},"sp":{"value":6102412368},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6594636288},"far":{"value":0}},"frames":[{"imageOffset":31232,"symbol":"kevent","symbolLocation":8,"imageIndex":14},{"imageOffset":336596,"symbol":"-[VDKQueue watcherThread:]","symbolLocation":260,"imageIndex":10},{"imageOffset":344052,"symbol":"__NSThread__start__","symbolLocation":716,"imageIndex":25},{"imageOffset":28724,"symbol":"_pthread_start","symbolLocation":136,"imageIndex":15},{"imageOffset":7740,"symbol":"thread_start","symbolLocation":8,"imageIndex":15}]},{"id":601150,"name":"com.apple.NSEventThread","threadState":{"x":[{"value":268451845},{"value":21592279046},{"value":8589934592},{"value":203439715909632},{"value":0},{"value":203439715909632},{"value":2},{"value":4294967295},{"value":18446744073709550527},{"value":47367},{"value":0},{"value":1},{"value":47367},{"value":522035},{"value":0},{"value":6597069766656},{"value":18446744073709551569},{"value":8190394416},{"value":0},{"value":4294967295},{"value":2},{"value":203439715909632},{"value":0},{"value":203439715909632},{"value":6107570280},{"value":8589934592},{"value":21592279046},{"value":21592279046},{"value":4412409862}],"flavor":"ARM_THREAD_STATE64","lr":{"value":6594686256},"cpsr":{"value":4096},"fp":{"value":6107570128},"sp":{"value":6107570048},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6594611380},"far":{"value":0}},"frames":[{"imageOffset":6324,"symbol":"mach_msg2_trap","symbolLocation":8,"imageIndex":14},{"imageOffset":81200,"symbol":"mach_msg2_internal","symbolLocation":80,"imageIndex":14},{"imageOffset":42224,"symbol":"mach_msg_overwrite","symbolLocation":476,"imageIndex":14},{"imageOffset":7224,"symbol":"mach_msg","symbolLocation":24,"imageIndex":14},{"imageOffset":515748,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":160,"imageIndex":20},{"imageOffset":509804,"symbol":"__CFRunLoopRun","symbolLocation":1208,"imageIndex":20},{"imageOffset":506972,"symbol":"CFRunLoopRunSpecific","symbolLocation":608,"imageIndex":20},{"imageOffset":1455108,"symbol":"_NSEventThread","symbolLocation":144,"imageIndex":22},{"imageOffset":28724,"symbol":"_pthread_start","symbolLocation":136,"imageIndex":15},{"imageOffset":7740,"symbol":"thread_start","symbolLocation":8,"imageIndex":15}]},{"id":984337,"frames":[{"imageOffset":7720,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":15}],"threadState":{"x":[{"value":6102986752},{"value":71127},{"value":6102450176},{"value":0},{"value":409604},{"value":18446744073709551615},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":0},"cpsr":{"value":4096},"fp":{"value":0},"sp":{"value":6102986752},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6594854440},"far":{"value":0}}},{"id":984338,"threadState":{"x":[{"value":4},{"value":0},{"value":0},{"value":601095},{"value":8352},{"value":6103553576},{"value":5536727168},{"value":6103549128},{"value":8022046047,"symbolLocation":39,"symbol":"sRMHandleArrayMutex"},{"value":8022046040,"symbolLocation":32,"symbol":"sRMHandleArrayMutex"},{"value":8022046032,"symbolLocation":24,"symbol":"sRMHandleArrayMutex"},{"value":984338},{"value":1},{"value":2},{"value":258},{"value":4373420000,"symbolLocation":0,"symbol":"OBJC_CLASS_$_NDResourceFork"},{"value":301},{"value":8190410088},{"value":0},{"value":8022046008,"symbolLocation":0,"symbol":"sRMHandleArrayMutex"},{"value":258},{"value":0},{"value":8022046032,"symbolLocation":24,"symbol":"sRMHandleArrayMutex"},{"value":984338},{"value":8022046040,"symbolLocation":32,"symbol":"sRMHandleArrayMutex"},{"value":0},{"value":105553145917968},{"value":7916784958,"objc-selector":"objectForKeyedSubscript:"},{"value":7916771949,"objc-selector":"boolValue"}],"flavor":"ARM_THREAD_STATE64","lr":{"value":6594863464},"cpsr":{"value":1610616832},"fp":{"value":6103553744},"sp":{"value":6103553696},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6594622820},"far":{"value":0}},"queue":"QSCatalogEntry scanQueue: QSPresetDesktop","frames":[{"imageOffset":17764,"symbol":"__psynch_mutexwait","symbolLocation":8,"imageIndex":14},{"imageOffset":16744,"symbol":"_pthread_mutex_firstfit_lock_wait","symbolLocation":84,"imageIndex":15},{"imageOffset":6904,"symbol":"_pthread_mutex_firstfit_lock_slow","symbolLocation":248,"imageIndex":15},{"imageOffset":15644,"symbol":"TSLockMutex","symbolLocation":176,"imageIndex":26},{"imageOffset":239104,"symbol":"GetRefHandle","symbolLocation":36,"imageIndex":26},{"imageOffset":256192,"symbol":"CloseResFileCommon","symbolLocation":320,"imageIndex":26},{"imageOffset":120040,"symbol":"-[NDResourceFork closeFile]","symbolLocation":32,"imageIndex":8},{"imageOffset":118240,"symbol":"-[NDAlias(AliasFile) initWithContentsOfFile:]","symbolLocation":116,"imageIndex":8},{"imageOffset":119184,"symbol":"+[NDAlias(AliasFile) aliasWithContentsOfFile:]","symbolLocation":36,"imageIndex":8},{"imageOffset":226864,"imageIndex":5},{"imageOffset":225152,"imageIndex":5},{"imageOffset":234040,"imageIndex":5},{"imageOffset":46724,"symbol":"-[QSCatalogEntry scannedObjects]","symbolLocation":76,"imageIndex":10},{"imageOffset":50440,"symbol":"__30-[QSCatalogEntry scanAndCache]_block_invoke","symbolLocation":32,"imageIndex":10},{"imageOffset":14608,"symbol":"_dispatch_client_callout","symbolLocation":20,"imageIndex":19},{"imageOffset":76996,"symbol":"_dispatch_lane_barrier_sync_invoke_and_complete","symbolLocation":56,"imageIndex":19},{"imageOffset":47292,"symbol":"-[QSCatalogEntry scanAndCache]","symbolLocation":208,"imageIndex":10},{"imageOffset":48104,"symbol":"-[QSCatalogEntry scanForced:]","symbolLocation":416,"imageIndex":10},{"imageOffset":469024,"symbol":"__CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__","symbolLocation":148,"imageIndex":20},{"imageOffset":1075436,"symbol":"___CFXRegistrationPost_block_invoke","symbolLocation":88,"imageIndex":20},{"imageOffset":1075252,"symbol":"_CFXRegistrationPost","symbolLocation":440,"imageIndex":20},{"imageOffset":267404,"symbol":"_CFXNotificationPost","symbolLocation":764,"imageIndex":20},{"imageOffset":36912,"symbol":"-[NSNotificationCenter postNotificationName:object:userInfo:]","symbolLocation":88,"imageIndex":25},{"imageOffset":332392,"symbol":"-[QSVoyeur queue:didReceiveNotification:forPath:]","symbolLocation":196,"imageIndex":10},{"imageOffset":338400,"symbol":"__26-[VDKQueue watcherThread:]_block_invoke","symbolLocation":228,"imageIndex":10},{"imageOffset":7352,"symbol":"_dispatch_call_block_and_release","symbolLocation":32,"imageIndex":19},{"imageOffset":14608,"symbol":"_dispatch_client_callout","symbolLocation":20,"imageIndex":19},{"imageOffset":86988,"symbol":"_dispatch_root_queue_drain","symbolLocation":864,"imageIndex":19},{"imageOffset":88580,"symbol":"_dispatch_worker_thread2","symbolLocation":156,"imageIndex":19},{"imageOffset":12504,"symbol":"_pthread_wqthread","symbolLocation":228,"imageIndex":15},{"imageOffset":7728,"symbol":"start_wqthread","symbolLocation":8,"imageIndex":15}]}],
  "usedImages" : [
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4554506240,
    "CFBundleShortVersionString" : "1.3.0",
    "CFBundleIdentifier" : "com.blacktree.Quicksilver.QSPrimerInterfacePlugIn",
    "size" : 16384,
    "uuid" : "4c7e4541-181f-308c-bf52-e6cbdba6744a",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/PlugIns\/Primer Interface.qsplugin\/Contents\/MacOS\/Primer Interface",
    "name" : "Primer Interface",
    "CFBundleVersion" : "191A"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4554424320,
    "CFBundleShortVersionString" : "3.1.0",
    "CFBundleIdentifier" : "com.blacktree.Quicksilver.QSHotKeyTriggerPlugIn",
    "size" : 16384,
    "uuid" : "aafd037b-b7c3-34b7-8cd9-85adc7ea8c21",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/PlugIns\/Keyboard Triggers.qsplugin\/Contents\/MacOS\/Keyboard Triggers",
    "name" : "Keyboard Triggers",
    "CFBundleVersion" : "1939"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 4551802880,
    "CFBundleShortVersionString" : "1.0",
    "CFBundleIdentifier" : "com.apple.icloud.drive.fileprovider.override",
    "size" : 16384,
    "uuid" : "32076e3f-af4f-3832-9a49-6552ddbcac62",
    "path" : "\/System\/Library\/Frameworks\/FileProvider.framework\/OverrideBundles\/iCloudDriveFileProviderOverride.bundle\/Contents\/MacOS\/iCloudDriveFileProviderOverride",
    "name" : "iCloudDriveFileProviderOverride",
    "CFBundleVersion" : "2461.40.47"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 4553146368,
    "CFBundleShortVersionString" : "14.1",
    "CFBundleIdentifier" : "com.apple.findersync.fileprovideroverride.FinderSyncCollaborationFileProviderOverride",
    "size" : 131072,
    "uuid" : "c69dfd08-9b3e-3e05-800d-c446e7e0502a",
    "path" : "\/System\/Library\/Frameworks\/FileProvider.framework\/OverrideBundles\/FinderSyncCollaborationFileProviderOverride.bundle\/Contents\/MacOS\/FinderSyncCollaborationFileProviderOverride",
    "name" : "FinderSyncCollaborationFileProviderOverride",
    "CFBundleVersion" : "1630.1.4"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 4550115328,
    "CFBundleShortVersionString" : "1703.41.2",
    "CFBundleIdentifier" : "com.apple.FileProviderOverride",
    "size" : 32768,
    "uuid" : "78f21b30-a3ca-3fc0-992f-a421a2e10051",
    "path" : "\/System\/Library\/Frameworks\/FileProvider.framework\/OverrideBundles\/FileProviderOverride.bundle\/Contents\/MacOS\/FileProviderOverride",
    "name" : "FileProviderOverride",
    "CFBundleVersion" : "1703.41.2"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4551901184,
    "CFBundleShortVersionString" : "1.3.0",
    "CFBundleIdentifier" : "com.blacktree.Quicksilver.QSCorePlugIn",
    "size" : 376832,
    "uuid" : "fa0b4288-55e7-3abe-aa4c-ae482e55ebc6",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/PlugIns\/Core Support.qsplugin\/Contents\/MacOS\/Core Support",
    "name" : "Core Support",
    "CFBundleVersion" : "99"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 4550246400,
    "CFBundleShortVersionString" : "3.0",
    "CFBundleIdentifier" : "com.apple.security.csparser",
    "size" : 131072,
    "uuid" : "af53e762-bef4-3d88-9797-ff28f3f85e76",
    "path" : "\/System\/Library\/Frameworks\/Security.framework\/Versions\/A\/PlugIns\/csparser.bundle\/Contents\/MacOS\/csparser",
    "name" : "csparser",
    "CFBundleVersion" : "61040.41.1"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 4370087936,
    "size" : 49152,
    "uuid" : "c9ce7923-2f2d-31eb-a6e4-0f3b7a78f895",
    "path" : "\/usr\/lib\/libobjc-trampolines.dylib",
    "name" : "libobjc-trampolines.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4373135360,
    "CFBundleIdentifier" : "com.blacktree.QSFoundation",
    "size" : 245760,
    "uuid" : "5ca56208-d6aa-3ede-a16c-2786258aa4eb",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/Frameworks\/QSFoundation.framework\/Versions\/A\/QSFoundation",
    "name" : "QSFoundation",
    "CFBundleVersion" : "2.3"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4372463616,
    "CFBundleIdentifier" : "com.blacktree.QSEffects",
    "size" : 49152,
    "uuid" : "86b20182-7aa2-3072-81ff-01c5dbfa7096",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/Frameworks\/QSEffects.framework\/Versions\/A\/QSEffects",
    "name" : "QSEffects",
    "CFBundleVersion" : "2.3"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4375183360,
    "CFBundleIdentifier" : "com.blacktree.QSCore",
    "size" : 442368,
    "uuid" : "56d32f7c-fbe1-3740-9e71-3ae69cde15c1",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/Frameworks\/QSCore.framework\/Versions\/A\/QSCore",
    "name" : "QSCore",
    "CFBundleVersion" : "2.3"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4372594688,
    "CFBundleIdentifier" : "com.blacktree.QSInterface",
    "size" : 229376,
    "uuid" : "23bfea87-78c4-3726-8c2a-10954e3c4ba2",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/Frameworks\/QSInterface.framework\/Versions\/A\/QSInterface",
    "name" : "QSInterface",
    "CFBundleVersion" : "2.3"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4372201472,
    "CFBundleShortVersionString" : "2.7.7",
    "CFBundleIdentifier" : "com.flyingmeat.FMDB-MacOS",
    "size" : 65536,
    "uuid" : "e8ff4f99-f16c-383f-87c9-10b6c4939472",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/Frameworks\/QSFoundation.framework\/Versions\/A\/Frameworks\/FMDB.framework\/Versions\/A\/FMDB",
    "name" : "FMDB",
    "CFBundleVersion" : "1"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4367319040,
    "CFBundleShortVersionString" : "2.4.1",
    "CFBundleIdentifier" : "com.blacktree.Quicksilver",
    "size" : 245760,
    "uuid" : "9bdac77f-8ca1-3cb9-9c33-f570bc7a48f0",
    "path" : "\/Applications\/Quicksilver.app\/Contents\/MacOS\/Quicksilver",
    "name" : "Quicksilver",
    "CFBundleVersion" : "4040"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6594605056,
    "size" : 241648,
    "uuid" : "b7751381-1442-30b5-91b9-ad7be461bebe",
    "path" : "\/usr\/lib\/system\/libsystem_kernel.dylib",
    "name" : "libsystem_kernel.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6594846720,
    "size" : 53236,
    "uuid" : "daf95373-5de6-39a1-a6ce-d87f3f0629cc",
    "path" : "\/usr\/lib\/system\/libsystem_pthread.dylib",
    "name" : "libsystem_pthread.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6593404928,
    "size" : 520188,
    "uuid" : "decb8685-f34a-3979-afcb-71fb55747e41",
    "path" : "\/usr\/lib\/system\/libsystem_c.dylib",
    "name" : "libsystem_c.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6594506752,
    "size" : 98304,
    "uuid" : "edbb6df0-72f9-3bc2-96fa-a86e05d3d305",
    "path" : "\/usr\/lib\/libc++abi.dylib",
    "name" : "libc++abi.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6590873600,
    "size" : 323337,
    "uuid" : "49e2dcb3-f014-3fcf-949b-f5f57b3ef0a8",
    "path" : "\/usr\/lib\/libobjc.A.dylib",
    "name" : "libobjc.A.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6593101824,
    "size" : 290816,
    "uuid" : "dc1d0187-7149-3100-bc63-f633afebee6c",
    "path" : "\/usr\/lib\/system\/libdispatch.dylib",
    "name" : "libdispatch.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6595264512,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.CoreFoundation",
    "size" : 5079040,
    "uuid" : "9f046e36-7286-3a6e-a280-699d6e47cfaf",
    "path" : "\/System\/Library\/Frameworks\/CoreFoundation.framework\/Versions\/A\/CoreFoundation",
    "name" : "CoreFoundation",
    "CFBundleVersion" : "2106"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6769098752,
    "CFBundleShortVersionString" : "2.1.1",
    "CFBundleIdentifier" : "com.apple.HIToolbox",
    "size" : 2883584,
    "uuid" : "4dffb7eb-bbcd-3ff6-a203-5a0b97358ee8",
    "path" : "\/System\/Library\/Frameworks\/Carbon.framework\/Versions\/A\/Frameworks\/HIToolbox.framework\/Versions\/A\/HIToolbox",
    "name" : "HIToolbox"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6654103552,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.AppKit",
    "size" : 20008960,
    "uuid" : "b475e541-691d-39c6-8ca4-e414b0d3dd88",
    "path" : "\/System\/Library\/Frameworks\/AppKit.framework\/Versions\/C\/AppKit",
    "name" : "AppKit",
    "CFBundleVersion" : "2487.20.107"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6591197184,
    "size" : 607000,
    "uuid" : "ec7a3ba0-f9bf-3ab8-a0f4-8622e5606b20",
    "path" : "\/usr\/lib\/dyld",
    "name" : "dyld"
  },
  {
    "size" : 0,
    "source" : "A",
    "base" : 0,
    "uuid" : "00000000-0000-0000-0000-000000000000"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6613262336,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.Foundation",
    "size" : 12902400,
    "uuid" : "2126fd9a-52ce-3eb2-8640-b57535e073b8",
    "path" : "\/System\/Library\/Frameworks\/Foundation.framework\/Versions\/C\/Foundation",
    "name" : "Foundation",
    "CFBundleVersion" : "2106"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6649061376,
    "CFBundleShortVersionString" : "1333",
    "CFBundleIdentifier" : "com.apple.CoreServices.CarbonCore",
    "size" : 3166208,
    "uuid" : "049ccd7b-4151-3816-8fa8-a0bad75ae69b",
    "path" : "\/System\/Library\/Frameworks\/CoreServices.framework\/Versions\/A\/Frameworks\/CarbonCore.framework\/Versions\/A\/CarbonCore",
    "name" : "CarbonCore",
    "CFBundleVersion" : "1333"
  }
],
  "sharedCache" : {
  "base" : 6590447616,
  "size" : 4018896896,
  "uuid" : "80dd42b3-8a52-3caf-9848-54a1c2732864"
},
  "vmSummary" : "ReadOnly portion of Libraries: Total=1.4G resident=0K(0%) swapped_out_or_unallocated=1.4G(100%)\nWritable regions: Total=2.2G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=2.2G(100%)\n\n                                VIRTUAL   REGION \nREGION TYPE                        SIZE    COUNT (non-coalesced) \n===========                     =======  ======= \nAccelerate framework               128K        1 \nActivity Tracing                   256K        1 \nColorSync                          576K       27 \nCoreAnimation                      384K       24 \nCoreGraphics                        48K        3 \nCoreServices                       624K        2 \nCoreUI image data                  128K        2 \nFoundation                          16K        1 \nImage IO                           176K        1 \nKernel Alloc Once                   32K        1 \nMALLOC                             2.1G       84 \nMALLOC guard page                  288K       18 \nMemory Tag 244                      16K        1 \nSQLite page cache                  384K        3 \nSTACK GUARD                       56.1M        5 \nStack                             10.1M        5 \nVM_ALLOCATE                        144K        9 \n__AUTH                            1669K      332 \n__AUTH_CONST                      27.5M      537 \n__CTF                               824        1 \n__DATA                            12.7M      536 \n__DATA_CONST                      27.3M      556 \n__DATA_DIRTY                      1705K      196 \n__FONT_DATA                          4K        1 \n__LINKEDIT                       887.5M       15 \n__OBJC_RO                         70.8M        1 \n__OBJC_RW                         2156K        1 \n__TEXT                           590.7M      573 \ndyld private memory                272K        2 \nlibnetwork                         640K       16 \nmapped file                      183.7M       41 \nshared memory                      896K       16 \n===========                     =======  ======= \nTOTAL                              4.0G     3012 \n",
  "legacyInfo" : {
  "threadTriggered" : {
    "queue" : "com.apple.main-thread"
  }
},
  "logWritingSignature" : "0b8afde2f1c1a8169db5bee78f0ca55ab3514645",
  "trialInfo" : {
  "rollouts" : [
    {
      "rolloutId" : "61675b89201f677a9a4cbd65",
      "factorPackIds" : {
        "HEALTH_FEATURE_AVAILABILITY" : "64d14f3e1008ca2fac17a163"
      },
      "deploymentId" : 240000064
    },
    {
      "rolloutId" : "5fb4245a1bbfe8005e33a1e1",
      "factorPackIds" : {

      },
      "deploymentId" : 240000021
    }
  ],
  "experiments" : [

  ]
}
}

**Comment by OlivierMartineau** [comment]
Same here: since Sonoma, Quicksilver is very often hanging on my MBP.
I also fear it is responsible for my keyboard input to freeze.

**Comment by alland138** [comment]
I’ve sent at least 10-15 crash logs over the last days - crashes now sometimes display them. But so far no word from anybody on what might be going on.

**Comment by n8henrie** [comment]
Sorry all. This is one of a long list of show-stopper Sonoma bugs. I'm the most junior member of the team and am trying to learn objc as I go, and I may be the only one running Sonoma, so things will be slow going for the foreseeable future.

**Comment by OlivierMartineau** [comment]
Thank you for replying. Let us know if we can do anything to help.

**Comment by alland138** [comment]
hi n8henri — thanks for your response. Totally understand — also available if I can be of help. Hopefully we can muddle along restarting as we go. Question — have you received the crash logs I sent in? Do you want me / us to continue forwarding them?

**Comment by n8henrie** [comment]
> Question — have you received the crash logs I sent in? Do you want me / us to continue forwarding them?

@alland138 The crash logs are anonymized, collected, and stored. I have 118 reports from Nov 2023 so far; I presume yours is one of them, but I'd need more info to be sure.

Continuing to send them would be great, thank you.

**Comment by alland138** [comment]
Hello Nathan

I imagine that something like 20% of those reports are from me … more will surely follow because QS crashes every hour or two, though there’s no discernible pattern to it

Best


Allan

______________________
 
￼

**Comment by n8henrie** [comment]
Would you mind telling me your computer name or hostname so I can try to find which crash logs are yours?

**Comment by n8henrie** [comment]
I don't see anything immediately obvious in your crash log above. 

Also, make sure to look through: https://docs.qsapp.com/documentation/faq#quicksilver-crashes-how-do-i-fix-it

My guess is that your crashes will get figured out much more quickly if you go through things yourself (and hopefully report back) -- clearing caches and disabling plugins is where I'd start.

**Comment by alland138** [comment]
Hi I just had a crash (in this case, the Apple crash log didn’t appear before the notification window to you was displayed (that happens sometimes — actually sometime no window at all appears and I just have to force quit.

To make it easy to identify my message, here’s what it contained. My exception is that you’ll be able to see identifying data in the data that accompanies the message and that will let you see all the logs I have sent:

Hi Nathan - here's the report from the latest crash — I'll send you a note now so that you can definitively identify my crash reports. In this case, though the crash log window didn't display, though i assume the data will come through to you directly anyway. Best, Allan

BTW, I have tried the cache cleaning and permissions fix routines repeatedly with Sonoma Cache Cleaner but no improvements from doing that. I’ll try the plug-ins but, frankly, I’m not overly hopeful .. still maybe I get lucky …




______________________
 
￼

**Comment by alland138** [comment]
Interesting — I’ve never seen other apps apparently impacted. Only symptoms I observe is QS becoming unresponsive and requiring a force quit then relaunch or QS crashing.




______________________
 
￼

**Comment by OlivierMartineau** [comment]
This has happened several times here.
I right click on QS icon in the menu bar, get nothing and basically loose the keyboard. Not all of it though, as I can still Cmd+Tab to switch apps.
It's not really impacting other apps, more like stilling the keyboard, which makes sense as QS is always listening for its keyboard shortcuts I guess.

I have now uninstalled QS as it was in the way big time, but totally regret having upgraded to Sonoma.

I use QS mainly for the the keyboard shortcuts to launch app though, and found another way to do so:
https://www.idownloadblog.com/2021/01/15/launch-app-keyboard-shortcut-mac/

**Comment by alland138** [comment]
Update — I updated to MacOS 14.1.2 — and around that time the continuous crashing seems to have stopped ... the funny thin is that I can’t say exactly when it stopped but obviously the lack of crash logs from me will confirm it. I also can’t confirm that the OS update brought the stability but what I do know is that I don’t have the continual crashing any more and there’s nothing that I can point to as the cause of the improvement. Hopefully this will last!

**Comment by OlivierMartineau** [comment]
Yes, the right (or left) click on the QS menu icon is the one on the upper bar, indeed. Makes QS steal my keyboard input.
It has never worked here since Sonoma. It was working perfectly before.

AppCleaner basically does what you describe. I have already lost my preferences indeed, but I'll be able to retrieve them if I ever want to on a backup.
I wanted to make sure it was not some settings that was bringing QS to its knees.

No clue what you line "Use Effects" does but it doesn't seem to change a thing here: QS fails the same.

**Comment by n8henrie** [comment]
Very weird. What interface are you using?

When you open Activity Monitor, is Quicksilver in red / marked as frozen?

Nate

**Comment by OlivierMartineau** [comment]
What do you mean by interface?

When I open Activity Monitor, Quicksilver is not marked as frozen.

**Comment by n8henrie** [comment]
Sorry, not sure what else to call it. The Bezel interface, or Primer, or Nostromo... the "theme" that determines how QS looks. NateOn Dec 5, 2023, at 6:26 AM, OM ***@***.***> wrote:﻿
What do you mean by interface?
When I open Activity Monitor, Quicksilver is not marked as frozen.

**Comment by OlivierMartineau** [comment]
Oh: didn't think the user input devices could have an impact.
I have a regular Apple USB-cable keyboard and a Logitech MX Ergo Graphite.
I'll check with another mouse, just in case.

**Comment by alland138** [comment]
Hi Nathan

For whatever reason QS now seems to be more stable — I had a freeze this morning (had to force quit) but that meant it had run without crashing since I last updated you. And so far, it’s gone about 12 hours without freezing or crashing completely  — so definitely major improvement in the situation.

Best


Allan




______________________
 
￼

**Comment by n8henrie** [comment]
> Oh: didn't think the user input devices could have an impact.

Not sure who you're responding to, but this is not what I was talking about. You can choose an interface in your appearance settings.

**Comment by OlivierMartineau** [comment]
Oh, I must have misread your post, sorry. I have no way to check this as the app won't respond as soon as I click on the icon in the menu bar :(

**Comment by alland138** [comment]
The crashing several times a day suddenly started again —- this is purely a subjective impression based on lots of trouble-shooting and QA experienced  .... I have not been able to repro it at will. But I’m starting to get the impression that the problem may be related to running WhatsApp Mac ... because using it more over the last days seems to have co-related with increased crashing. I’ll keep an eye out and perhaps the others on this thread can confirm whether their experience is in some way similar ...

**Comment by OlivierMartineau** [comment]
Since [MacOS Sonoma update](https://support.apple.com/fr-fr/HT214048)  on Dec. 19th Quicksilver icon as reappeared in the menu bar. And it works perfectly so far, I'm very happy :)

I now understand what Bezer/Primer refers to, by the way, and QS is set to Primer apparently here.

**Comment by n8henrie** [comment]
> And it works perfectly so far, I'm very happy :)

Well that's a relief!

**Comment by edwardhorsford** [comment]
With Sonoma 14.2 on a Macbook Pro M2, I'm finding quicksilver has either crashed or self-quit about 1-2 times per day at the moment. Wishing you luck in diagnosing this.

**Comment by anatomatic** [comment]
I'm one of those submitting crash reports, sometimes multiple a day. QS always seems to self-quit when I'm away from the computer — on wake? on sleep — it rarely happens when I'm working. I have an automation in Keyboard Maestro that checks if Quicksilver is running on wake and if not, relaunches. Let me know if you can use any more info to help diagnose.

**Comment by alland138** [comment]
Hi — would you mind sharing how you configured the Keyboard Maestro automation you mentioned in your posting? I might get it to help cope with the crashing (which I wish was only once a day … the number of daily crashes/freezes has increased again, it seems). Thanks




______________________
 
￼

**Comment by anatomatic** [comment]
This image shows the KM automation and you can download [it here](https://www.dropbox.com/scl/fi/hyk6otwdtus99b5403ejw/QS-launch-on-wake.kmmacros?rlkey=wbrveqrfb80057weuk1ity2is&dl=0)

![QS-launch-on-wake](https://github.com/quicksilver/Quicksilver/assets/463305/d2aad390-7db4-4a05-bb04-cbb66c6c48c3)

**Comment by alland138** [comment]
Brilliant — thanks very much

I'll install KM and get this going.Itll save lots of frustration as a workaround till the bug is swatted.




______________________
 
￼

**Comment by piramasa** [comment]
In my environment, installing the Safari plugin eats up all the memory.
QS: 2.4.2, M1 MacBookPro, MacOS is Sonoma (14.2.1).

**Comment by n8henrie** [comment]
@alland138 @OlivierMartineau @alland138

I just updated and released a new version of the Safari plugin that should resolve a bug that caused it to consume a mountain of energy, leading to lag and even crashes for some users. Can you please update and report back if it resolves your issue?

I will likely close this issue if @alland138 either reports back and is fixed or doesn't report back. Others, if still having issues please start a new issue so we can collect the data we require to debug!

**Comment by alland138** [comment]
Hi — I’m afraid that I can’t be of help any more as I was forced to migrate to Keyboard Maestro … but despite that, thanks for posing the update!

A




				______________________

 

￼

**Comment by n8henrie** [comment]
No problem, thanks for the update!
